### PR TITLE
[Test] qol vehicle tile changes

### PR DESCRIPTION
**Pull request by RepresentativeGear9**
#### Summary
NeoDays "QoL vehicle tile changes"

#### Content of the change

2391_vp_stowboard_vertical_0

### CODE REVIEW

**Comment by int-ua** [comment]
Errors:
```
ERROR process_png: duplicate root name 702_seat_0: gfx/NeoDays/pngs_tiles_10x10/vehicle/702_seat_0.png
ERROR convert: vp_door_trunk_broken encountered more than once, last time in gfx/NeoDays/pngs_tiles_10x10/vehicle/vp_door_trunk_vp_hatch_vp_door_shutter_vp_door_sliding_0.json
ERROR convert: vp_door_trunk_open encountered more than once, last time in gfx/NeoDays/pngs_tiles_10x10/vehicle/vp_door_trunk_vp_hatch_vp_door_shutter_vp_door_sliding_0.json
ERROR convert: vp_door_trunk encountered more than once, last time in gfx/NeoDays/pngs_tiles_10x10/vehicle/vp_door_trunk_vp_hatch_vp_door_shutter_vp_door_sliding_0.json
ERROR convert: vp_door_shutter encountered more than once, last time in gfx/NeoDays/pngs_tiles_10x10/vehicle/vp_door_trunk_vp_hatch_vp_door_shutter_vp_door_sliding_0.json
```

**Comment by int-ua** [comment]
Differences in IDs don't look right:
```
-vp_door_sliding
+vp_door_sliding_0
+3857_seat_0
```
Looks like you've used sprite names instead of IDs somewhere

File: gfx/NeoDays/pngs_tiles_10x10/vehicle/vp_door_sliding_0.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"id": "vp_door_sliding_0", "fg": ["3855_vp_door_sliding_0"], "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": ["2286_vp_door_trunk_1"], "bg": []}, {"id": "open", "fg": ["3855_vp_door_sliding_1"], "bg": []}], "bg": []}
```
Comment (int-ua) [inline gfx/NeoDays/pngs_tiles_10x10/vehicle/vp_door_sliding_0.json:1]:
```suggestion
{"id": "vp_door_sliding", "fg": ["3855_vp_door_sliding_0"], "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": ["2286_vp_door_trunk_1"], "bg": []}, {"id": "open", "fg": ["3855_vp_door_sliding_1"], "bg": []}], "bg": []}
```

**Comment by int-ua** [comment]
And the `3857_seat_0.png` is not used anywhere so a tile entry for it was created. But that is an incorrect ID.

File: gfx/NeoDays/pngs_tiles_10x10/vehicle/vp_door_sliding_0.json (hunk 1 above)
Comment (RepresentativeGear9) [reply gfx/NeoDays/pngs_tiles_10x10/vehicle/vp_door_sliding_0.json:1]: Ahh must have misplaced that "_0". and yes, i forgot to change the seat json to use my sprite

**Comment by RepresentativeGear9** [comment]
> And the `3857_seat_0.png` is not used anywhere so a tile entry for it was created. But that is an incorrect ID.

or did you mean i cant use "3857" in front of my sprite name? 0_0

**Comment by int-ua** [comment]
Closing per discussion at Discord, the fork will be created from scratch and with feature branches this time :)

**Pull request closed without merging**